homeassistant/components/device_tracker/bluetooth_tracker.py

```diff
@@ -0,0 +1,90 @@
+"""Tracking for bluetooth devices."""
+import logging
+import os
+from datetime import timedelta
+
+from homeassistant.helpers.event import track_point_in_utc_time
+from homeassistant.components.device_tracker import (
+    YAML_DEVICES,
+    CONF_TRACK_NEW,
+    DEFAULT_CONF_TRACK_NEW,
+    CONF_SCAN_INTERVAL,
+    DEFAULT_SCAN_INTERVAL,
+)
+from homeassistant.config import load_yaml_config_file
+import homeassistant.util as util
+import homeassistant.util.dt as dt_util
+
+_LOGGER = logging.getLogger(__name__)
+
+REQUIREMENTS = ['pybluez==0.22']
+
+
+def setup_scanner(hass, config, see):
+    """Setup the Bluetooth Scanner."""
+    import bluetooth
+
+    def see_device(device, see):
+        """Mark a device as seen."""
+        see(dev_id='bt_' + device[1], mac=device[0], host_name=device[1])
+
+    def discover_devices():
+        """Discover bluetooth devices."""
+        result = bluetooth.discover_devices(duration=8,
+                                            lookup_names=True,
+                                            flush_cache=True,
+                                            lookup_class=False)
+        _LOGGER.info("Bluetooth devices discovered = " + str(len(result)))
+        return result
+
+    yaml_path = hass.config.path(YAML_DEVICES)
```

Comment:
Device Tracker has a helper method for this: [`load_config`](https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/device_tracker/__init__.py#L371)